Code/Mantid/Framework/Algorithms/src/PlotAsymmetryByLogValue.cpp

```diff
@@ -322,6 +213,51 @@ void PlotAsymmetryByLogValue::exec() {
   setProperty("OutputWorkspace", outWS);
 }
 
+/**  Loads one run and applies dead-time corrections and detector grouping if required
+*   @param runNumber :: [input] Run number specifying run to load
+*/
+Workspace_sptr PlotAsymmetryByLogValue::doLoad (int64_t runNumber ) {
+
+  // Get complete run name
+  std::ostringstream fn, fnn;
+  fnn << std::setw(m_filenameZeros) << std::setfill('0') << runNumber;
+  fn << m_filenameBase << fnn.str() << m_filenameExt;
+
+  // Load run
+  IAlgorithm_sptr load = createChildAlgorithm("LoadMuonNexus");
+  load->setPropertyValue("Filename", fn.str());
+  load->execute();
+  Workspace_sptr loadedWs = load->getProperty("OutputWorkspace");
+
+  // Check if dead-time corrections have to be applied
+  if (m_dtcType != "None") {
+    if (m_dtcType == "FromSpecifiedFile") {
+
+      // If user specifies a file, load corrections now
+      Workspace_sptr customDeadTimes;
+      if (m_dtcType == "FromSpecifiedFile") {
```

Comment:
This second `if` statement is identical the the first, so it'll probably cause a Coverity issue. Removing it would also make the logic here a little tidier.